lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -2454,19 +2463,30 @@ balanceTransactionSpec = describe "balanceTransaction" $ do
         it "roundtrips with toCardanoValue" $
             property prop_posAndNegFromCardanoValueRoundtrip
 
-    describe "effect of txMaxSize on coin selection" $ do
-        let addr = Address $ unsafeFromHex
-                "60b1e5e0fb74c86c801f646841e07\
-                \cdb42df8b82ef3ce4e57cb5412e77"
-        let mw = SomeMnemonic $ either (error . show) id
-                (entropyToMnemonic @12 <$> mkEntropy "0000000000000000")
-        let rootK = Shelley.unsafeGenerateKeyFromSeed (mw, Nothing) mempty
+    it "increases zero-ada outputs to minimum" $ do
+        let era = WriteTx.RecentEraBabbage
+        let out = TxOut dummyAddr (TokenBundle.fromCoin (Coin 0))
+        let out' = TxOut dummyAddr (TokenBundle.fromCoin (Coin 874_930))
```

Comment:
Hmm. Where did the `874_930` value come from?

Is it just picked randomly to be somewhat lower than the minimum required value?

Reply:
This is the computed (safe) minimum. Using the alonzo computation.

Reply:
No, right this is Babbage. Because it's lower than what is seen in `MinimumUTxOSpec` because the dummy address here only contains a payment key (without the extra 28 bytes for delegation key).

```
995610-(28*34482/8) = 874923
```